addons/web/static/tests/core/model_field_selector_tests.js

```diff
@@ -720,4 +741,91 @@ QUnit.module("Components", (hooks) => {
             target.querySelector(".o_model_field_selector_popover_search .o_input")
         );
     });
+
+    QUnit.test("support properties", async (assert) => {
+        addProperties();
+
+        class Parent extends Component {
+            static components = { ModelFieldSelector };
+            static template = xml`
+                <ModelFieldSelector
+                    readonly="false"
+                    resModel="'partner'"
+                    path="path"
+                    isDebugMode="true"
+                    update="(path, fieldInfo) => this.onUpdate(path)"
+                />
+            `;
+            setup() {
+                this.path = "foo";
+            }
+            onUpdate(path) {
+                this.path = path;
+                assert.step(path);
+                this.render();
+            }
+        }
+
+        await mountComponent(Parent);
+        await openModelFieldSelectorPopover(target);
+        assert.strictEqual(getTitle(target), "");
+        assert.containsOnce(target, ".o_model_field_selector_popover_item[data-name='properties']");
+        assert.containsOnce(
+            target,
+            ".o_model_field_selector_popover_item[data-name='properties'] .o_model_field_selector_popover_relation_icon"
+        );
+        assert.deepEqual(getModelFieldSelectorValues(target), ["Foo"]);
+        assert.containsNone(target, ".o_model_field_selector_warning");
+
+        await click(
+            target,
+            ".o_model_field_selector_popover_item[data-name='properties'] .o_model_field_selector_popover_relation_icon",
+            "click on the relation icon should open the properties page"
+        );
+        assert.deepEqual(getModelFieldSelectorValues(target), ["Properties"]);
+        assert.containsOnce(target, ".o_model_field_selector_warning");
+        assert.verifySteps([]);
+
+        await openModelFieldSelectorPopover(target);
```

Comment:
not useful I think

Reply:
or not very natural

Reply:
it is, it closes the popover and re-opens it on the first page
I change the line to `await clickPrev(target);`